frontend/src/components/RecordsList.tsx

```diff
@@ -200,24 +200,23 @@ const RecordsList: FunctionComponent<IRecordsListProps> = ({
         </Modal>
       )}
 
-      {sampleQueryParamValue && (
+      {samplesQueryParam && (
         <AutoSizer>
           {({ height, width }) => (
             <Modal show={true} dialogClassName="modal-90w" onHide={handleClose}>
               <Modal.Header closeButton>
-                <Modal.Title>{`Viewing ${sampleQueryParamValue}`}</Modal.Title>
+                <Modal.Title>{`Viewing ${samplesQueryParam}`}</Modal.Title>
               </Modal.Header>
               <Modal.Body>
-                <div style={{ height: 600 }}>
+                <div className={styles.popupHeight}>
```

Comment:
This is a minor change that sets the AG Grid table height to fill up the available space, removing the unused gap below the table.